osmdroid-android/src/main/java/org/osmdroid/tileprovider/tilesource/ITileSource.java

```diff
@@ -39,11 +39,13 @@
 	/**
 	 * Get a rendered Drawable from the specified file path.
 	 *
+	 * @param aTile
+	 * 						a tile
 	 * @param aFilePath
 	 *            a file path
 	 * @return the rendered Drawable
 	 */
-	Drawable getDrawable(String aFilePath) throws LowMemoryException;
+	Drawable getDrawable(MapTile aTile, String aFilePath) throws LowMemoryException;
```

Comment:
this part erks me a bit. Is there a way to do perform the same function without modifying the interface? there's a ton of tile sources out there. merging this would cause compatibility issues